src/app/geo-header/geo-header.component.ts

```diff
@@ -28,4 +28,23 @@ export class GeoHeaderComponent {
       .dispatch(changeSearchFromAddress($value));
   }
 
+  geolocate() {
+    this.store
```

Comment:
Ok with the first part of my comment.

The second part refers to the fact that the component is making use of the service layer directly, instead of dispatching a geolocating action to the store. Why is it?